roles/zabbix_web/defaults/main.yml

```diff
@@ -28,15 +28,15 @@ zabbix_php_fpm_conf_listen: true
 # zabbix_web_tls_key: /etc/pki/server.key
 # zabbix_web_tls_chain:
 # zabbix_web_SSLPassPhraseDialog: exec:/usr/libexec/httpd-ssl-pass-dialog
-# zabbix_web_SSLSessionCache: shmcb:/run/httpd/sslcache(512000)
-# zabbix_web_SSLSessionCacheTimeout: 300
 # zabbix_web_SSLCryptoDevice: builtin
 # zabbix_web_max_execution_time: 300
 # zabbix_web_memory_limit: 128M
 # zabbix_web_post_max_size: 16M
 # zabbix_web_upload_max_filesize: 2M
 # zabbix_web_max_input_time: 300
 # zabbix_web_max_input_vars: 10000
+zabbix_web_ssl_session_cache_timeout: "{{ zabbix_web_SSLSessionCacheTimeout | default(omit) }}"  # Remove backwards compatability in 4.0
+zabbix_web_ssl_session_cache: "{{ zabbix_web_SSLSessionCache | default(omit) }}"  # Remove backwards compatability in 4.0
```

Comment:
This use of omit is no good. Omit just spits out a placeholder with some random garbage. So if you're not tweaking the values, you'll end up with something like this;

```
SSLSessionCache __omit_place_holder__08d954833cbf3b3f396e4cf23c4715143f9d0299
SSLSessionCacheTimeout __omit_place_holder__08d954833cbf3b3f396e4cf23c4715143f9d0299
```

Defaults are there to provide values, and otherwise best left commented out. (You can get around some of this by setting them to null, but that results in very un-ansible looking code with all these extra checks, and you're gonna have a bad time.)

Reply:
Don't love this fix but it works